debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleConnectorConfig.java

```diff
@@ -51,46 +51,8 @@
 
     protected static final int DEFAULT_PORT = 1528;
 
-    public static final Field HOSTNAME = Field.create(DATABASE_CONFIG_PREFIX + JdbcConfiguration.HOSTNAME)
-            .withDisplayName("Hostname")
-            .withType(Type.STRING)
-            .withWidth(Width.MEDIUM)
-            .withImportance(Importance.HIGH)
-            .withValidation(OracleConnectorConfig::requiredWhenNoUrl)
```

Comment:
I don't believe we can use the `RelationalDatabaseConnectorConfig` definition as-is since the Oracle implementation has a special validation that allows the `database.hostname` to be omitted when a URL is provided.  

```suggestion
    public static final Field HOSTNAME = RelationalDatabaseConnectorConfig.HOSTNAME
            .withNoValidation()
            .withValidation(OracleConnectorConfig::requiredWhenNoUrl);          
```

The remaining references to `RelationalDatabaseConnectorConfig.HOSTNAME` should then be changed to use the connector specific HOSTNAME reference instead.

Reply:
I added a commit that satisfies this requirement, tests now all pass.